packages/cloud-cognitive/src/components/Datagrid/Datagrid/DraggableElement.js

```diff
@@ -11,10 +11,12 @@ import PropTypes from 'prop-types';
 import { Draggable16 } from '@carbon/icons-react';
 import { useDrag, useDrop } from 'react-dnd';
 import cx from 'classnames';
+import { pkg } from '../../../settings';
 
 const { useEffect, useRef, useState } = React;
+const blockClass = `${pkg.prefix}--datagrid`;
 
-const DRAG_TYPE = 'wkc-shared-ui-draggable-element';
+const DRAG_TYPE = `${blockClass}__shared-ui-draggable-element`;
```

Comment:
I can't find a corresponding class name in `draggableElement.scss`. Is the style defined somewhere else?